backend/copr-backend.spec

```diff
@@ -1,7 +1,3 @@
-%if 0%{?rhel} <= 7 && 0%{?rhel} > 0
```

Comment:
I don't think we build copr-backend for RHEL.  Seems safe to abandon %py3_* scripts entirely here.

Reply:
Ok. updated.